### DISCUSSION TITLE
[DONE] Release to crates.io and package name

### DISCUSSION THREAD

**Discussion by arnauorriols** [iotaledger/identity.rs#663] [Technical Discussion]
Are you planning to release identity.rs to crates.io? And assuming you are, under which package name? [identity is already taken](https://crates.io/crates/identity)

**Comment by olivereanderson** [comment]
Hey @arnauorriols,
 identity.rs is on crates.io now. The crate name is [identity_iota](https://crates.io/crates/identity_iota).
